src/journeys/feedbackService.js

```diff
@@ -0,0 +1,53 @@
+function validateFeedback(req, res) {
+  try {
+    const { feedbackEntity, feedbackType, feedbackDescription, feedbackConsent } = req.body;
+
+    let showFeedbackTypeError = false;
+    let showFeedbackDescriptionError = false;
+    let showFeedbackConsentError = false;
+    let showFeedbackEntityError = false;
+
+    //Check feedbackEntity field value
+    if (!feedbackEntity || feedbackEntity === "") {
+      // Check if in edit mode and redirect accordingly
+      showFeedbackEntityError = true;
+    }
+
+    //Check feedbackType field value
+    if (!feedbackType || feedbackType === "") {
+      showFeedbackTypeError = true;
+    }
+
+    //Check feedbackDescription field value
+    if (!feedbackDescription || feedbackDescription === "") {
+      showFeedbackDescriptionError = true;
+    }
+
+    if (!feedbackConsent || feedbackConsent === "") {
```

Comment:
Minor one, for consistency lets add the below?
//Check feedbackConsent field value

its obvious but just for consistency cause the others have that too